tsconfig.json

```diff
@@ -6,9 +6,12 @@
     "declarationDir": "dist",
     "esModuleInterop": true,
     "jsx": "react-jsx",
+    "jsxImportSource": "@emotion/react",
+    "lib": ["dom", "es2017"],
     "noEmit": true,
     "skipLibCheck": true,
-    "strict": true
+    "strict": true,
+    "types": ["jest", "jest-enzyme", "jest-plugin-context"]
```

Comment:
Oddly, I could not get some of `jest-enzyme`'s matchers to type-check consistently without adding `jest-enzyme` to `types` here.  And now that we're specifying `types` explicitly, we had to add `jest` and `jest-plugin-context` here as well, along with `"lib": ["dom", "es2017"]` above.